include/dbclean.php

```diff
@@ -21,6 +22,10 @@ function dbclean_run(&$argv, &$argc) {
 	load_config('config');
 	load_config('system');
 
+	if (!get_config("system", "dbclean")) {
```

Comment:
Oh, that sounds like another code-convention. Maybe I can handle them, if the approach of @rabuzarus is wanted?